romfs/source/fighter/bayonetta/param/vl.prcxml

```diff
@@ -17,6 +17,11 @@
     <float hash="final_vector_min">40</float>
     <float hash="final_vector_max">45</float>
   </struct>
+  <list hash="param_specialn_bullet">
+	  <struct index="0">
+		  <int hash="0x0d76ae442e">3</int>
```

Comment:
I would suggest not using `tab` to space out lines in the prcxml, use spaces instead